src/Pages/SignUp/SignUpInput.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import styled from "styled-components";
+
+const SignUpInput = props => {
```

Comment:
```suggestion
const SignUpInput = ({ data }) => {
```
함수 컴포넌트의 props 자리에서 직접 구조분해할당 해줄 수 있습니다

Reply:
넵 반영하겠습니다! 

const { data } = props; 으로 구조분해할당할 수 있는거는 이해가 되었는데
⚠️ props 자리에 ({ data }) 를 넣을 수 있는 이유 찾아보기 
sconst SignupInout = ({ data }) => {